src/Tizen.NUI/src/public/WebView/WebViewPageLoadErrorEventArgs.cs

```diff
@@ -129,10 +130,11 @@ public string PageUrl
         }
 
         private LoadErrorCode _errorCode;
+
         /// <summary>
         /// The code for the current error.
         /// </summary>
-        [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Please do not open setter of this property.
How about make setter to internal?

Reply:
done.